lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -2352,30 +2372,32 @@ defmodule Kernel.SpecialForms do
   Any new and existing messages that do not match will remain in the mailbox.
 
   ## Examples
-
-      receive do
-        {:selector, number, name} when is_integer(number) ->
-          name
-        name when is_atom(name) ->
-          name
-        _ ->
-          IO.puts(:stderr, "Unexpected message received")
-      end
+      iex> send(self(), {:selector, 5, :quantity})
+      iex> receive do
+      ...>   {:selector, number, name} when is_integer(number) ->
+      ...>     name
+      ...>   name when is_atom(name) ->
+      ...>     name
+      ...>   _ ->
+      ...>     IO.puts(:stderr, "Unexpected message received")
+      ...> end
+      :quantity
 
   An optional `after` clause can be given in case no matching message is
   received during the given timeout period, specified in milliseconds:
 
-      receive do
-        {:selector, number, name} when is_integer(number) ->
-          name
-        name when is_atom(name) ->
-          name
-        _ ->
-          IO.puts(:stderr, "Unexpected message received")
-      after
-        5000 ->
-          IO.puts(:stderr, "No message in 5 seconds")
-      end
+      iex> receive do
+      ...>   {:selector, number, name} when is_integer(number) ->
+      ...>     name
+      ...>   name when is_atom(name) ->
+      ...>     name
+      ...>   _ ->
+      ...>     IO.puts(:stderr, "Unexpected message received")
+      ...> after
+      ...>   10 ->
+      ...>     "return this after 10 milliseconds"
```

Comment:
I think the original message was nicer to explain?

```suggestion
      ...>     "No message in 10 milliseconds"
```